.travis.yml

```diff
@@ -0,0 +1,40 @@
+
+branches:
+ only:
+  - master
+
+language: python
+
+matrix:
+  include:
+    - os: linux
+      python: "3.4"
+
+    - os: linux
+      python: "3.3.5"
+
+    - os: linux
+      python: "2.7"
+
+cache:
+  directories:
+    - $HOME/.pyenv/versions/3.5.2
+    - $HOME/.pyenv/versions/3.4.5
+    - $HOME/.pyenv/versions/3.3.6
+    - $HOME/.pyenv/versions/2.7.12
+    - $HOME/downloads
```

Comment:
These directories look suspicious to me. Furthermore, the versions do not match the ones configured above?